content/kubermatic/main/tutorials-howtos/OIDC-Provider-Configuration/_index.en.md

```diff
@@ -102,10 +102,26 @@ spec:
       }
 ```
 
+#### Seed Configuration
+
+In some cases a Seed may require an independent OIDC provider. For this reason a `Seed` CRD contains relevant fields under `spec.oidcProviderConfiguration`. Filling those results in overwriting a configuration from `KubermaticConfiguration` CRD. The following snippet presents an example of `Seed` CRD configuration:
```

Comment:
```
Filling those results in overwriting a...
```
This sentence was confusing to me because "results" can be verb or plural noun. Maybe it can be changed to:
```
Filling those fields results in overwriting a...
```